### PR TITLE
Job tag characters

### PR DESCRIPTION
**Pull request by djyasin**
##### SUMMARY

Revised job_tags to handle more than 1024 characters

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - API

##### AWX VERSION

```
awx: 21.0.1
```

### CODE REVIEW

File: awx/api/serializers.py [hunk 1]
```diff
@@ -4480,7 +4480,10 @@ def __init__(self, *args, **kwargs):
                 body = messages[event].get('body', {})
                 if body:
                     try:
-                        potential_body = json.loads(body)
+                        rendered_body = (
+                            sandbox.ImmutableSandboxedEnvironment(undefined=DescriptiveUndefined).from_string(body).render(JobNotificationMixin.context_stub())
+                        )
+                        potential_body = json.loads(rendered_body)
```
Comment (AlanCoding) [inline awx/api/serializers.py]: looks like this got copied over from another branch

Comment (djyasin) [reply awx/api/serializers.py]: Oh no!! Thank you so much for spotting that!

Comment (djyasin) [reply awx/api/serializers.py]: @AlanCoding I believe I have successfully deleted the change to serializers.py that was added by mistake.

**Comment by shanemcd** [comment]
The commit history here looks incorrect. See if a rebase helps. Feel free to reach out if you need any help.

**Review by AlanCoding — APPROVED** [review]

**Review by john-westcott-iv — APPROVED** [review]

**Review by shanemcd — APPROVED** [review]

**Comment by jay-steurer** [comment]
Tested and Approved.

**Pull request merged**